### ISSUE TITLE
Develop / Document a process for landing revisions

### ISSUE THREAD

**Issue by ericlucit** [willson556/phorge-devcontainer#4]
I think there are a couple of things to work out related to landing revisions

1.  By default the repos are cloned via HTTPS - In order to land (push) you will have to add a vcs user and pass to the origin url
2. If we change to SSH urls (recommended) - What public key can we use? - Would have to generate a public key manually and then upload this to your phorge account

**Comment by willson556** [comment]
Yeah, I'm not really sure what the right path is here. I think guiding the user towards SSH is probably correct.

The question then is do we recommend:

1. Generate public key, upload to Phorge account, then update `origin`
 -- OR --
2. Map `~/.ssh` in the dev container to a location configured by environment variable (defaulting to `~/.ssh` on the host)

 Option 2 is probably the most convenient but it's probably not best practice to expose that location to docker. So probably option 1?

We could include a `configure_git_ssh.sh` script in the container that looks something like
```sh
#! /bin/bash

PHORGE_URL="ssh://git@we.phorge.it/source/phorge.git"
ARCANIST_URL="ssh://git@we.phorge.it/source/arcanist.git"

pushd /srv/phorge/phorge
git remote set-url origin $PHORGE_URL
popd

pushd /srv/phorge/arcanist
git remote set-url origin $ARCANIST_URL
popd

echo "Generate a key at the default location:"
ssh-keygen -t ed25519 -C "phorge-devcontainer-$HOSTNAME"

echo "Upload the following public key to http://we.phorge.it/settings/panel/ssh :"
cat ~/.ssh/id_ed25519.pub
```

**Comment by ericlucit** [comment]
I think the script is a good approach.  Because then we can add a simple step in the README.

And I `think` that this key will persist (unless the container is completely rebuilt?)

Or - We should document the cases in which the key would be to be regenerated.

I think also, there is the conduit token.   So, maybe the script also includes `arc install-certificate` ?
